### PR TITLE
feat: update de.csv

### PR DESCRIPTION
**Pull request by wojosc**
Especially to allign Sales Orders (Aufträge) Sales Invoices (Ausgangsrechnungen) to Purchase Orders (Bestellungen) Purchase Invoice (Eingangsrechnungen). Both are a definate opposite and are kept short.

> no-docs

### CODE REVIEW

File: erpnext/translations/de.csv [hunk 1]
```diff
@@ -7518,27 +7518,27 @@ Timeline,Zeitleiste,
...
 Task Depends On,Vorgang hängt ab von,
-Task Type,Aufgabentyp,
+Task Type,Vorgangstyp,
 TS-.YYYY.-,TS-.YYYY.-,
 Employee Detail,Mitarbeiterdetails,
 Billing Details,Rechnungsdetails,
-Total Billable Hours,Insgesamt abrechenbare Stunden,
-Total Billed Hours,Insgesamt Angekündigt Stunden,
+Total Billable Hours,Summe abrechenbare Stunden,
+Total Billed Hours,Summe Angekündigt Stunden,
```
Comment (barredterra) [inline erpnext/translations/de.csv]:
```suggestion
Total Billed Hours,Summe abgerechneter Stunden,
```

File: erpnext/translations/de.csv [hunk 2]
```diff
@@ -7518,27 +7518,27 @@ Timeline,Zeitleiste,
...
 Total Costing Amount,Gesamtkalkulation Betrag,
-Total Billable Amount,Insgesamt abrechenbare Betrag,
+Total Billable Amount,Summe abrechenbare Betrag,
```
Comment (barredterra) [inline erpnext/translations/de.csv]:
```suggestion
Total Billable Amount,Summe abrechenbarer Betrag,
```

File: erpnext/translations/de.csv [hunk 3]
```diff
@@ -7555,13 +7555,13 @@ Quality Goal,Qualitätsziel,
...
-Quality Goal Objective,Qualitätsziel Ziel,
+Quality Goal Objective,Qualitätsziel,
 Objective,Zielsetzung,
 Agenda,Agenda,
-Minutes,Protokoll,
+Minutes,Protokolle,
 Quality Meeting Agenda,Qualitätstreffen Agenda,
 Quality Meeting Minutes,Qualitätssitzungsprotokoll,
-Minute,Minute,
+Minute,Protokoll,
```
Comment (barredterra) [inline erpnext/translations/de.csv]: Needs context. "Minute" (as a unit of time) would also be correct.

File: erpnext/translations/de.csv [hunk 4]
```diff
@@ -7845,13 +7845,13 @@ Bank Balance,Kontostand,
...
 Purchase Orders to Bill,Bestellungen an Rechnung,
-New Sales Orders,Neue Kundenaufträge,
+New Sales Orders,Neue Aufträge,
 New Purchase Orders,Neue Bestellungen an Lieferanten,
-Sales Orders to Deliver,Kundenaufträge zu liefern,
-Purchase Orders to Receive,Bestellungen zu empfangen,
-New Purchase Invoice,Neue Kaufrechnung,
+Sales Orders to Deliver,Auszuliefernde Aufträge,
+Purchase Orders to Receive,Anzuliefernde Bestellungen,
+New Purchase Invoice,Neue Ausgangsrechnung,
```
Comment (barredterra) [inline erpnext/translations/de.csv]:
```suggestion
New Purchase Invoice,Neue Eingangsrechnung,
```

File: erpnext/translations/de.csv [hunk 5]
```diff
@@ -7972,7 +7972,7 @@ MAT-DN-.YYYY.-,MAT-DN-.YYYY.-,
 Is Return,Ist Rückgabe,
 Issue Credit Note,Gutschrift ausgeben,
 Return Against Delivery Note,Zurück zum Lieferschein,
-Customer's Purchase Order No,Kundenauftragsnr.,
+Customer's Purchase Order No,Bestell Nr. des Kunden,
```
Comment (barredterra) [inline erpnext/translations/de.csv]:
```suggestion
Customer's Purchase Order No,Bestellnummer des Kunden,
```

**Review by barredterra — CHANGES_REQUESTED** [review]

File: erpnext/translations/de.csv (hunk 3 above)
Comment (wojosc) [reply erpnext/translations/de.csv]: > Needs context. "Minute" (as a unit of time) would also be correct.

Both occour. Minute das the time factor and minute as a type of meeting. No idea which one will be which :/

File: erpnext/translations/de.csv (hunk 1 above)
Comment (wojosc) [reply erpnext/translations/de.csv]: > 

Maybe sogar "Summe angekündigter Stunden" ?

Comment (barredterra) [reply erpnext/translations/de.csv]: Wieso angekündigt?

File: erpnext/translations/de.csv (hunk 3 above)
Comment (barredterra) [reply erpnext/translations/de.csv]: We'd need to [add context](https://frappeframework.com/docs/v13/user/en/translations#6-adding-context-for-a-string) to the translatable strings in the source code.

Comment (barredterra) [reply erpnext/translations/de.csv]:
```suggestion
Minute,Minute,
```

Let's skip this for now and get the PR merged

**Review by barredterra — APPROVED** [review]

**Pull request merged**